docs/content/core/volumes.md

```diff
@@ -213,5 +214,115 @@ fin config set DOCKSAL_VOLUMES=none
 fin project reset
 ```
 
-Use `fin bash` to log into bash and checkout files into `/var/www` with git. Or use `fin docker cp` to copy 
-files into the `cli` container. 
+Use `fin bash` to log into bash and checkout files into `/var/www` with git. Or use `fin docker cp` to copy
+files into the `cli` container.
+
+### Mutagen {#volumes-mutagen}
+
+- This option only makes sense with Docker Desktop on macOS.
+- Like the Unison option, Mutagen is used to synchronize the files from host to the container.
+- There are two configuration and usage options for Mutagen: Docker Desktop Extension and Stand alone.
+
+#### The Docker Desktop Extension
+
+- This option is more recent but comes in two flavors
+- The Free version which only allows for one Cache Setup.
+- The Paid Subscription ( https://mutagen.io/account/subscription $7/month) which allows for multiple cache directory setups.
```

Comment:
This should be...
- [The Paid Subscription](https://mutagen.io/account/subscription $7/month) which allows for multiple cache directory setups.